build.gradle

```diff
@@ -124,16 +124,16 @@ publishing {
 
 dependencies {
     implementation 'com.google.protobuf:protobuf-lite:3.0.1'
-    implementation 'androidx.core:core:1.0.0'
-    implementation 'com.google.android.material:material:1.2.0-rc01'
+    implementation 'androidx.core:core:1.7.0'
+    implementation 'com.google.android.material:material:1.6.0-alpha02'
```

Comment:
i think we should stick with the latest stable version instead of using an alpha version

Reply:
Thank you Caren, the version changed to 1.5.0.